backend/app2/llm/tests/test_anthropic_adapter.py

```
@@ -0,0 +1,215 @@
+import pytest
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)